test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -16,12 +16,15 @@ describe('Deployments View widget', () => {
     const blueprintName = `${specPrefix}blueprint`;
     const deploymentId = `${specPrefix}deployment_id`;
     const deploymentName = `${specPrefix}deployment_name`;
+    const labelName = `${specPrefix}label_name`;
+    const labelVal = `${specPrefix}label_val`;
```

Comment:
More accurate with the resource naming convention would be:
```suggestion
    const labelKey = `${specPrefix}label_name`;
    const labelValue = `${specPrefix}label_val`;
```

WDYT?